common/artifact_spawns/wc_artifact_spawns.txt

```diff
@@ -485,4 +485,317 @@ aluneth = {
     artifacts = {
 		aluneth = { value = 1 }
     }
+}
+felomelorn = {
+	max_amount = 1
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 42002		# Anasterian
+		}
+	}
+    artifacts = {
+		felomelorn = { value = 1 }
+    }
+}
+hammer_of_the_twilight = {
+	max_amount = 1
+    spawn_date = 1.1.583
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+		modifier = {
+			factor = 0
+			NOT = { year = 585 }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			is_playable = yes
+			
+			culture = twilights_hammer
+		}
+	}
+    artifacts = {
+		hammer_of_the_twilight = { value = 1 }
+    }
+}
+hammer_of_the_lightbringer = {
+	max_amount = 1
+    spawn_date = 1.1.583
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 60036						# Uther Lightbringer
+		}
+	}
+    artifacts = {
+		hammer_of_the_lightbringer = { value = 1 }
+    }
+}
+tabard_of_the_silver_hand = {
+	max_amount = 1
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = xxxxx						# all five paladins that are created by event
```

Comment:
Then you don't need this thing here

Reply:
Oh right, I oversaw it after event was edited.

Reply:
deleted